### DISCUSSION TITLE
What is your highlighting system?

### DISCUSSION THREAD

**Discussion by krystoflp** [BryanJenksCommunity/FAQ#137] [Q&A] [answered]
Hi Brian, Thanks so much for all the obsidian insight. You've really helped me make a transistion to Obsidian, and I'm so glad I did.

I have one query from your videos and your templates about your highlighting system.

Most of your highlighting color suggestions use two colors separated by an arrow.  

This is the example from your tempates that I'm thinking of:

Highlight Colors --> Thanks [Bri Watson](app://obsidian.md/@Brian%20Watson)!
Yellow --> Orange --> Important Point By Author
Pink --> Red --> Disagree With Author
Green --> Green --> Important To Me
Purple --> Magenta --> Literary Note To Lookup Later

Please could you explain how you differentiate between the two colors on each line?

**Comment by tallguyjenks** [comment, marked as answer]
the different colors are just the colors that the Highlights mac app pdf reader uses and when i extract annotations using the zotero workflow, they get extracted to the 2nd column of colors

**Comment by krystoflp** [comment]
Thank you Brian for explaining this :-)---- On Thu, 17 Feb 2022 05:09:37 +0000  ***@***.******@***.***> wrote ---- 
the different colors are just the colors that the Highlights mac app pdf reader uses and when i extract annotations using the zotero workflow, they get extracted to the 2nd column of colors
